Sources/MapboxMaps/Annotations/AnnotationOrchestrator.swift

```diff
@@ -49,24 +64,28 @@ public protocol AnnotationInteractionDelegate: AnyObject {
 /// `AnnotationOrchestrator` provides a way to create annotation managers of different types.
 public class AnnotationOrchestrator {
 
-    private let gestureRecognizer: UIGestureRecognizer
+    private let tapGestureRecognizer: UIGestureRecognizer
+
+    private let longPressGestureRecognizer: UIGestureRecognizer
 
     private let style: Style
 
     private let mapFeatureQueryable: MapFeatureQueryable
 
     private weak var displayLinkCoordinator: DisplayLinkCoordinator?
 
-    internal init(gestureRecognizer: UIGestureRecognizer,
+    internal init(tapGestureRecognizer: UIGestureRecognizer,
+                  longPressGestureRecognizer: UIGestureRecognizer,
                   mapFeatureQueryable: MapFeatureQueryable,
                   style: Style,
                   displayLinkCoordinator: DisplayLinkCoordinator) {
-        self.gestureRecognizer = gestureRecognizer
+        self.tapGestureRecognizer = tapGestureRecognizer
+        self.longPressGestureRecognizer = longPressGestureRecognizer
         self.mapFeatureQueryable = mapFeatureQueryable
         self.style = style
         self.displayLinkCoordinator = displayLinkCoordinator
 
-        gestureRecognizer.addTarget(self, action: #selector(handleTap(_:)))
+        tapGestureRecognizer.addTarget(self, action: #selector(handleTap(_:)))
```

Comment:
we previously have a tap gesture that is handled in `AnnotationOrchestrator`, from there we would call to each registered annotation manages to handle each gesture's state. Could we keep the same pattern here for consistency?